### DISCUSSION TITLE
Size of a Maui app is twice the size of a xamarin app

### DISCUSSION THREAD

**Discussion by developer9969** [dotnet/maui#20681] [Q&A]
Hi wondering if anyone reached the same conclusions the size of both android and iOS is way bigger than the equivalent in xamarin.
I have build the app in Maui with the same settings as in xamarin same cpu architecture and I get a way bigger app. I am talking release mode.

Where is the bloat?
Any suggestions on optimisation of settings in case I am missing 
something?

Thanks for any suggestions

**Comment by VNGames** [comment]
Same here. Just finished migrating apps. Almost twice the size.

**Comment by indieshack** [comment]
Ditto.
